omod/src/main/webapp/resources/htmlforms/diabetic_clinical_follow_up.xml

```diff
@@ -1235,28 +1235,118 @@
+                   </tr>
+                   <tr>
+                       <td>
+                           <label> (i) Urea (mmol/l)</label>
+                       </td>
+                       <td>
+                           <obs id="urea" conceptId="53874292-7a10-4726-bf30-64513c9e774a" dateLabel="&lt;/td&gt;&lt;td&gt;" />
+                       </td>
+                   </tr>
+                   <tr>
+                       <td colspan="3" >
+                           <label>Electrolytes</label>
+                       </td>
+                   </tr>
+                   <tr>
+                       <td>
+                           <label>Sodium (mmol/l)</label>
+                       </td>
+                       <td>
+                           <obs id="sodium" conceptId="b29c84d6-7625-4ff2-9d8f-646820a318bc" dateLabel="&lt;/td&gt;&lt;td&gt;" />
+                       </td>
+                   </tr>
+                   <tr>
+                       <td>
+                           <label>Chloride (mmol/l)</label>
+                       </td>
+                       <td>
+                           <obs id="chloride" conceptId="60c4617e-67ce-4824-b67d-f0b51fa742a2" dateLabel="&lt;/td&gt;&lt;td&gt;" />
+                       </td>
+                   </tr>
+                   <tr>
+                       <td>
+                           <label>Potassium (mmol/l)</label>
+                       </td>
+                       <td>
+                           <obs id="potassium" conceptId="2e6af16c-d013-4c81-9764-b1166e3e0e57" dateLabel="&lt;/td&gt;&lt;td&gt;" />
+                       </td>
+                   </tr>
+                   <tr>
+                       <td>
+                           <label>Creatinine (mmol/l)</label>
+                       </td>
+                       <td>
+                           <obs id="Creatinine" conceptId="164364AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" dateLabel="&lt;/td&gt;&lt;td&gt;" />
+                       </td>
+                   </tr>
+
+                   <tr>
+                       <td>
+                           <span class="sub-title">Urinalysis:</span>
+                           <table>
+                               <tbody>
+                                   <tr>
+                                       <td>
+                                           <label>Glucose</label>
+                                       </td>
+                                       <td>
+                                           <obs id="glucose" conceptId="159733" answerConceptIds="1066,1065" answerLabels="No,Yes" style="radio" class="align-radio-input" answerSeparator="" />
+                                           <hr />
+                                           <span id="glucose-level">
+                                               <obs conceptId="159733" answerConceptIds="1362,1363,1364" answerLabels="+,++,+++" style="radio" class="align-radio-input" answerSeparator=""/>
```

Comment:
It would be great if you use concept uuids as oppossed to concept ids. This helps during data migration and harmonization